### PR TITLE
Support unoptimized InterchainGasPaymaster bytecode

### PR DESCRIPTION
**Pull request by nambrot**
### Description

Testnet4 has unoptimized Bytecode that did not get added as a constant in the checker

### Drive-by changes

CLI test was referencing the wrong folder

### CODE REVIEW

**Comment by jmrossy** [comment]
@nambrot Is this still relevant?

**Comment by nambrot** [comment]
Yessir, @yorhodes can you approve?

File: typescript/cli/ci-test.sh [hunk 1]
```diff
@@ -105,7 +105,7 @@ yarn workspace @hyperlane-xyz/cli run hyperlane send transfer \
 MESSAGE2_ID=`cat /tmp/message2 | grep "Message ID" | grep -E -o '0x[0-9a-f]+'`
 echo "Message 2 ID: $MESSAGE2_ID"
 
-cd ./rust
+cd ../../rust
```
Comment (yorhodes) [inline typescript/cli/ci-test.sh:108]: how is this relevant?

**Review by yorhodes — APPROVED** [review]

File: typescript/cli/ci-test.sh (hunk 1 above)
Comment (nambrot) [reply typescript/cli/ci-test.sh]: Mistaken change
